packages/theming/theme-tokens/src/dark/reactnative/tokens-global.json

```diff
@@ -1,833 +1,1004 @@
+      "shade10": "#106ebe",
+      "shade20": "#005a9e",
+      "shade30": "#004c87",
+      "shade40": "#004578",
+      "shade50": "#043862",
+      "shade60": "#092c47",
+      "tint10": "#2899f5",
+      "tint20": "#3aa0f3",
+      "tint30": "#6cb8f6",
+      "tint40": "#c7e0f4",
+      "tint50": "#deecf9",
+      "tint60": "#eff6fc"
+    },
+    "brandShadowAmbient": "#0000004d",
+    "brandShadowKey": "#00000040",
+    "brass": {
+      "primary": "#986f0b",
+      "shade10": "#89640a",
+      "shade20": "#745408",
+      "shade30": "#553e06",
+      "shade40": "#2e2103",
+      "shade50": "#181202",
+      "tint10": "#a47d1e",
+      "tint20": "#b18c34",
+      "tint30": "#c1a256",
+      "tint40": "#e0cea2",
+      "tint50": "#efe4cb",
+      "tint60": "#fbf8f2"
+    },
+    "bronze": {
+      "primary": "#a74109",
+      "shade10": "#963a08",
+      "shade20": "#7f3107",
+      "shade30": "#5e2405",
+      "shade40": "#321303",
+      "shade50": "#1b0a01",
+      "tint10": "#b2521e",
+      "tint20": "#bc6535",
+      "tint30": "#ca8057",
+      "tint40": "#e5bba4",
+      "tint50": "#f1d9cc",
+      "tint60": "#fbf5f2"
+    },
+    "brown": {
+      "primary": "#8e562e",
+      "shade10": "#804d29",
+      "shade20": "#6c4123",
+      "shade30": "#50301a",
+      "shade40": "#2b1a0e",
+      "shade50": "#170e07",
+      "tint10": "#9c663f",
+      "tint20": "#a97652",
+      "tint30": "#bb8f6f",
+      "tint40": "#ddc3b0",
+      "tint50": "#edded3",
+      "tint60": "#faf7f4"
+    },
+    "burgundy": {
+      "primary": "#a4262c",
+      "shade10": "#942228",
+      "shade20": "#7d1d21",
+      "shade30": "#5c1519",
+      "shade40": "#310b0d",
+      "shade50": "#1a0607",
+      "tint10": "#af393e",
+      "tint20": "#ba4d52",
+      "tint30": "#c86c70",
+      "tint40": "#e4afb2",
+      "tint50": "#f0d3d4",
+      "tint60": "#fbf4f4"
+    },
+    "charcoal": {
+      "primary": "#393939",
+      "shade10": "#333333",
+      "shade20": "#2b2b2b",
+      "shade30": "#202020",
+      "shade40": "#111111",
+      "shade50": "#090909",
+      "tint10": "#515151",
+      "tint20": "#686868",
+      "tint30": "#888888",
+      "tint40": "#c4c4c4",
+      "tint50": "#dfdfdf",
+      "tint60": "#f7f7f7"
+    },
+    "cornflower": {
+      "primary": "#4f6bed",
+      "shade10": "#4760d5",
+      "shade20": "#3c51b4",
+      "shade30": "#2c3c85",
+      "shade40": "#182047",
+      "shade50": "#0d1126",
+      "tint10": "#637cef",
+      "tint20": "#778df1",
+      "tint30": "#93a4f4",
+      "tint40": "#c8d1fa",
+      "tint50": "#e1e6fc",
+      "tint60": "#f7f9fe"
+    },
+    "cranberry": {
+      "primary": "#c50f1f",
+      "shade10": "#b10e1c",
+      "shade20": "#960b18",
+      "shade30": "#6e0811",
+      "shade40": "#3b0509",
+      "shade50": "#200205",
+      "tint10": "#cc2635",
+      "tint20": "#d33f4c",
+      "tint30": "#dc626d",
+      "tint40": "#eeacb2",
+      "tint50": "#f6d1d5",
+      "tint60": "#fdf3f4"
```

Comment:
I count 13. Is this the same as the web unified one we mentioned earlier?

Reply:
Yeah, this takes from the web tokens so it has 13 for the brand entry